databases/week1/hyf-week1.sql

```diff
@@ -0,0 +1,51 @@
+USE `hyf-week1`;
+ 
+SELECT * FROM `task`;
+SELECT * FROM `user`;
+SELECT * FROM `status`;
+SELECT `status`. `name` FROM `task`;
+SELECT * FROM `task` WHERE status_id != 3;
+-- Find out how many tasks are in the task table
+SELECT COUNT(id) FROM `task`;
+
+-- Find out how many tasks in the task table do not have a valid due date
+SELECT * FROM `task`WHERE due_date IS NULL;
+
+-- Find all the tasks that are marked as done
+SELECT * FROM `task` WHERE status_id = 3;
+
+-- Find all the tasks that are not marked as done
+SELECT * FROM `task` WHERE status_id < 3;
+
+-- Get all the tasks, sorted with the most recently created first
+SELECT * FROM `task` ORDER BY due_date DESC;
+
+-- Get the single most recently created task
+SELECT * FROM `task` ORDER BY due_date DESC LIMIT 1;
+
+-- Get the title and due date of all tasks where the title or description contains database
+SELECT `title`, `due_date`
+FROM `task`
+WHERE `title` 
+LIKE '%database%' 
+OR `description` 
+LIKE '%database%';
+
+-- Get the title and status (as text) of all tasks
+SELECT title, status_id AS 'text'
+FROM `task`;
```

Comment:
This can maybe help you to turn the desired result "task status as a text....Done, In progress or Not started"
```
SELECT title, status.name
AS 'text' FROM `task`
join status on status.id=task.status_id
ORDER BY status.name;
```